mandatory/1-fix-functions.js

```diff
@@ -10,8 +10,8 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(moodCondition) {
+  let isHappy = moodCondition;
```

Comment:
Given that you're assigning the value of one variable to another, can you think of a way to simplify this?